app/jobs/report_attachment_job.rb

```diff
@@ -0,0 +1,7 @@
+class ReportAttachmentJob < ApplicationJob
```

Comment:
@brianjaustin Would there be performance problems with this and numerous/large downloads? I'm not very familiar with async jobs.

Reply:
My initial thought was that we'll be fine, especially since HTML downloads require the least resources to generate. I double checked with the rest of Systems to be sure, and we're good in terms of workers for nwo